x-pack/plugins/security_solution/public/detections/components/value_lists_management_modal/flyout.tsx

```diff
@@ -92,10 +91,10 @@ export const ValueListsModalComponent: React.FC<ValueListsModalProps> = ({
 
   const handleReferenceDelete = useCallback(async () => {
     setShowReferenceErrorModal(false);
-    deleteList({ deleteReferences: true, http, id: referenceModalState.valueListId });
-    setReferenceModalState(referenceModalInitialState);
+    deleteList({ deleteReferences: true, http, id: referenceFlyoutState.valueListId });
+    setReferenceFlyoutState(referenceModalInitialState);
```

Comment:
```suggestion
    setReferenceFlyoutState(referenceFlyoutInitialState);
```

Reply:
This one (the reference modal) is still a modal that pops out so gonna leave that one.